src/app/pages/FrontPage/index.test.jsx

```diff
@@ -163,7 +163,7 @@ describe('Front Page', () => {
       expect(langSpan.getAttribute('lang')).toEqual('en-GB');
       expect(langSpan.textContent).toEqual('BBC News');
 
-      await wait();
+      await waitFor(() => {});
```

Comment:
`waitFor()` now requires a callback even if it is empty.

Reply:
Could you check if these `wait`s are really necessary?

Reply:
will double check

Reply:
The wait is needed here due to some async updates that happen within the toggles containers. A future PR could refactor this to use `act(() =>{})` to handle those updates.

Reply:
@hindsc52 Thanks for this PR. Could you please write up an issue to move from using waitFor() to act() for all relevant tests in the codebase?

Reply:
https://github.com/bbc/simorgh/issues/6384